pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "booster"
+version = "0.1.0"
+description = "A CLI tool for generating Django projects for Ackama."
+readme = "README.md"
+requires-python = ">=3.12"
+license = {text = "BSD-3-Clause"}
+dependencies = [
+    "typer[all]>=0.9.0",
+    "jinja2>=3.1.0",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** I don't think jinja2 is needed any more

If we don't need it - please remove.
‏

Reply:
Whoops!